apps/api/src/app/inbox/utils/notification-mapper.ts

```diff
@@ -35,8 +35,8 @@ const mapSingleItem = ({
     // subject: subject,
     body: content as string,
     to,
-    read,
-    archived,
+    isRead: read,
+    isArchived: archived,
```

Comment:
updated these fields because we want to have the functions on the Notification class with the names: `read, archive`